src/utils/channel_utils.js

```diff
@@ -5,8 +5,12 @@ import {General, Preferences} from 'constants';
 import {displayUsername} from './user_utils';
 import {getPreferencesByCategory} from './preference_utils';
 
-const defaultPrefix = 'D'; // fallback for future types
-const typeToPrefixMap = {[General.OPEN_CHANNEL]: 'A', [General.PRIVATE_CHANNEL]: 'B', [General.DM_CHANNEL]: 'C', [General.GM_CHANNEL]: 'C'};
+const channelTypeOrder = {
+    [Constants.OPEN_CHANNEL]: 0,
```

Comment:
Constants is not defined, I think it has to be `General`

Reply:
Ah, that's what I get for copying and pasting from the webapp repo